CHANGELOG

```diff
@@ -1,49 +1,70 @@
+Release 79 Black Swan (2016-04-xx)
+--------------------------------------
+CHANGELOG: removed closes from tickets in R78 (#2534)
+Scala Common Enrich: bumped user-agent-utils version to latest (#2516)
+Scala Common Enrich: transaction item quantity type changed to JInteger (#2157)
+Scala Hadoop Shred: bumped to 0.9.0 (#2480)
+Scala Hadoop Shred: bumped Scala Common Enrich to 0.23.0 (#2481)
+Scala Hadoop Shred: bumped Iglu Scala Client to 0.4.0 (#2449)
+Scala Hadoop Enrich: bumped to 1.7.0 (#2446)
+Scala Hadoop Enrich: bumped Scala Common Enrich to 0.23.0 (#2485)
+Scala Hadoop Enrich: bumped Iglu Scala Client to 0.4.0 (#2478)
+Common: changed Vagrantfile to use NFS and extra CPU cores by default (#2482)
+Config: removed duplicated enabled property in ua_parser_config.json (#2424)
+Config: enabled switched to false in currency_conversion_config.json (#2327)
+Config: enabled switched to false in weather_enrichment_config.json (#2326)
+Scala Common Enrich: bumped to 0.23.0 (#2486)
+Scala Common Enrich: improved OWM error if user doesn't have historical weather (#2325)
+Scala Common Enrich: added API Request Enrichment (#2051)
+Scala Common Enrich: bumped Iglu Scala Client to 0.4.0 (#2333)
+Scala Common Enrich: added HTTP Header Extractor Enrichment (#1373)
```

Comment:
Reorder this CHANGELOG into: Documentation; Common; Config; Scala Common Enrich; Scala Hadoop Enrich; Scala Hadoop Shred. It will flow much better.